### ISSUE TITLE
Make accessibility improvements default

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#109]
Some of the accessibility improvements introduced in Trix 3.0 were made opt-ins (#90); it should simply be the default.

**Comment by torgeirl** [comment]
Very poor description by me. :woman_facepalming: 

What I meant was that the toogle should be removed and everyone should be served the same, accessible content instead of leaving it to the users to opt-in or opt-out.

As far as I know there isn't anything in the accessibility improvements we need to tone down / adjust when showing it to all users.

**Comment by Levijatan** [comment]
applied the wcag fixes on building css.

**Comment by torgeirl** [comment]
Solved with `3.0.0b13`.
